docs/Oracle Jobs/tasks.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: nodes.liquid
+date: Last Modified
+title: "Tasks"
+permalink: "docs/tasks/"
+whatsnext: {"Introduction to External Adapters":"/docs/external-adapters/", "Jobs":"/docs/jobs/"}
+---
+
+## What is a Task?
+
```

Comment:
I think the new version of tasks has to be updated on a few other pages:
- [Fulfilling requests](/docs/fulfilling-requests/#add-jobs-to-the-node)
- [Bridges](/docs/node-operators/)